developer/revisions.md

```diff
@@ -1,102 +1,3 @@
-            "handle" : "example-product",
-            "hashtags" : [
-                "rpjCvTBGjhBi2xdro",
-                "cseCBSSrJ3t8HQSNP"
-            ],
-            "price" : {
-                "range" : "12.99 - 19.99",
-                "min" : 12.99,
-                "max" : 19.99
-            },
-            "isVisible" : true,
-            "isLowQuantity" : false,
-            "isSoldOut" : false,
-            "isBackorder" : false,
-            "metafields" : [
-                {
-                    "key" : "Material",
-                    "value" : "Cotton"
-                },
-                {
-                    "key" : "Quality",
-                    "value" : "Excellent"
-                }
-            ],
-            "pageTitle" : "This is a basic product. You can do a lot with it.",
-            "type" : "simple",
-            "updatedAt" : ISODate("2016-11-30T16:27:56.853Z"),
-            "vendor" : "Example Manufacturer",
-            "requiresShipping" : true,
-            "isDeleted" : false,
-            "workflow" : {
-                "status" : "new"
-            }
-        },
-        "createdAt" : ISODate("2016-11-30T16:27:56.857Z"),
-        "updatedAt" : ISODate("2016-11-30T16:32:17.469Z"),
-        "workflow" : {
-            "status" : "revision/published"
-        },
-        "diff" : [
-            {
-                "kind" : "E",
-                "path" : [
-                    "title"
-                ],
-                "lhs" : "Basic Reaction Product",
-                "rhs" : "Reaction Product"
-            },
-            {
-                "kind" : "E",
-                "path" : [
-                    "updatedAt"
-                ],
-                "lhs" : ISODate("2014-06-01T19:17:13.949Z"),
-                "rhs" : ISODate("2016-11-30T16:27:56.853Z")
-            }
-        ]
-    }
-```
-
-## Helpers
-
-### Check if revision control is enabled
-
-```js
-    import { isRevisionControlEnabled } from "/imports/plugins/core/revisions/lib/api";
-
-    if (isRevisionControlEnabled()) {
-      // Do something if revision control is enabled
-    }
-```
+> _Deprecation warning:_ Revision Control has been deprecated from Reaction as of 1.13. The legacy revision control system has been deprecated in favor of the new Catalog collection for overall product gird performance improvements and simplifed product publication logic. Read more about the changes in the CHANGELOG and [here](https://github.com/reactioncommerce/reaction/pull/4238).
```

Comment:
Same deal as above, `[here]` should say `[pull request]` or something to be clear where you're going.